internal/btesting/btesting.go

```diff
@@ -66,7 +75,24 @@ func MustOpenDBWithOption(t testing.TB, f string, o *bolt.Options) *DB {
 	}
 	resDB.strictModeEnabledDefault()
 	t.Cleanup(resDB.PostTestCleanup)
-	return resDB
+	return resDB, nil
+}
+
+// Creates a new database size, forces a specific allocation size jump, and fills it with the number of keys specified
+func CreateFilledDB(t testing.TB, o *bolt.Options, allocSize int, numKeys int) *DB {
+	// Open a data file.
+	db := MustCreateDBWithOption(t, o)
+	db.AllocSize = allocSize
+
+	// Insert a reasonable amount of data below the max size.
+	err := db.Fill([]byte("data"), 1, numKeys,
+		func(tx int, k int) []byte { return []byte(fmt.Sprintf("%04d", k)) },
+		func(tx int, k int) []byte { return make([]byte, 1000) },
+	)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
ditto, pls put such hard code in test cases instead of in the btesting.go
```suggestion
```

Reply:
ok